components/resume/job/job.tsx

```diff
@@ -1,10 +1,12 @@
 import React, { FC } from 'react';
 import { Job } from '../types';
 import Position from './position';
+import DateRange from './dateRange/dateRange';
 
-const Job: FC<Job> = ({ companyName, positions, highlights }) => {
+const Job: FC<Job> = ({ companyName, positions, highlights, startDate, endDate }) => {
     return <div>
         <h3>{companyName}</h3>
+        <div><DateRange start={startDate} end={endDate} /></div>
```

Comment:
This is to show a total duration at a job, makes more sense when there are multiple positions at a single company. For now this looks a bit odd for jobs with a single position, ie 
![image](https://user-images.githubusercontent.com/667983/159184885-128cfb2b-653e-43ef-b180-c9aaf6b1f8e1.png)

Leaving it as is for now and will defer crating a follow up task till after I tackle the style issue. Will need to figure out how to handle this type of scenario visually.